domain/infosync/tiflash_manager.go

```diff
@@ -69,8 +71,62 @@ func (m *TiFlashPDPlacementManager) Close(ctx context.Context) {
 
 }
 
+// SetTiFlashGroupConfig sets the tiflash's rule group config
+func (m *TiFlashPDPlacementManager) SetTiFlashGroupConfig(ctx context.Context) error {
```

Comment:
It is fine to just override the old group config. There is no need to `get&put`. BTW, `override: true` is needed to make `Index` effective.

Reply:
In the future the definition of `placement.RuleGroupConfig` changes, we can avoid modify other fields. We only need to set index and override fields.